### PR TITLE
test: zombie_task_kill

### LINKED ISSUE #61: Rewrite tests from tap to luatest
https://github.com/tarantool/expirationd/pull/52#discussion_r654379246

### PR DESCRIPTION
**Pull request by ArtDu**
Transferring the taptest to the luatest testing system with renaming the tests and using the existing spaces in helpers.

Check if the task will restart correctly if it sleeps for a long `full_scan_delay`.

Updated test's names:

- kill zombie test -> test_zombie_task_kill

Closes #61

### CODE REVIEW

File: test/unit/update_and_kill_test.lua [hunk 1]
```diff
@@ -94,3 +99,37 @@ function g.test_expirationd_update(cg)
         end)
     end
 end
+
+function g.test_zombie_task_kill(cg)
+    local space = cg.space
+    local task_name = cg.task_name
```
Comment (oleg-jukovec) [inline test/unit/update_and_kill_test.lua:105]:
```suggestion
    local space = 'test'
    local task_name = cg.first_space
```

File: test/unit/update_and_kill_test.lua [hunk 2]
```diff
@@ -26,6 +26,11 @@ g.before_each({index_type = 'HASH'}, function(cg)
     cg.second_space = second_space
 end)
 
+g.before_each(function(cg)
+    cg.task_name = 'test'
+    cg.space = cg.first_space
+end)
```
Comment (oleg-jukovec) [inline test/unit/update_and_kill_test.lua:32]:
```suggestion
```

**Comment by oleg-jukovec** [comment]
Let's do this way because it doesn't change anything, but there are fewer confusing variables in `cg`.

**Review by GRISHNOV — APPROVED** [review]

**Review by oleg-jukovec — APPROVED** [review]
Thank you very much!

**Pull request merged**